config/creatures/castle.json

```diff
@@ -346,11 +346,6 @@
 		"faction": "castle",
 		"abilities":
 		{
-			"KING_2" : // Will be affected by Advanced Slayer or better
```

Comment:
Does this mean that the fields in json now have to follow a certain order?

Reply:
Just for reading convenience.
Game does not retains this information at all, it is all lost during json parsins

Reply:
_it is all lost during json parsing_

*unfortunately it is all lost during json parsing. :)

Hopefully sometime in the future the order could be preserved. The information would be presented more player friendly.